spring-cloud-generator/sanity-checks.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+WORKING_DIR=`pwd`
```

Comment:
Please explain in the comments at the top of the script how to use it. Which scripts is it testing?

Reply:
Added overview of the script